src/jvmMain/kotlin/com/github/doyaaaaaken/kotlincsv/client/CsvWriter.kt

```diff
@@ -73,24 +96,59 @@ actual class CsvWriter actual constructor(
         return CsvFileWriter(ctx, PrintWriter(osw))
     }
 
+    /**
+     * *** ONLY for long-running write case ***
+     *
+     * Get and use [CsvFileWriter] directly.
+     * MUST NOT forget to close [CsvFileWriter] after using it.
+     *
+     * Use this method If you want to close file writer manually (i.e. streaming scenario).
+     */
+    @KotlinCsvExperimental
+    suspend fun openAndGetRawWriterAsync(ops: OutputStream): CsvFileWriter = withContext(Dispatchers.IO) {
```

Comment:
This method is also experimental, so remove it please.

Reply:
Removed on [0bd6bad](https://github.com/doyaaaaaken/kotlin-csv/pull/73/commits/0bd6bad257581a147721b4665fe7271515a90eff)